autoload/compe/documentation.vim

```diff
@@ -55,24 +69,31 @@ function! compe#documentation#open(document) abort
     endif
 
     let l:size = s:window.get_size({
-    \   'maxwidth': float2nr(&columns * 0.4),
+    \   'maxwidth': float2nr(&columns * 0.4) - 2,
     \   'maxheight': float2nr(&lines * 0.3),
     \ })
+    let l:size.width += 2 " padding
 
     let l:pos = s:get_screenpos(l:state.pumpos, l:size)
     if empty(l:pos)
-      return s:window.close()
+      return compe#documentation#close()
     endif
 
-    if pumvisible()
-      silent call s:window.open({
-      \   'row': l:pos[0] + 1,
-      \   'col': l:pos[1] + 1,
-      \   'width': l:size.width,
-      \   'height': l:size.height,
-      \ })
-      silent call s:Window.do(s:window.get_winid(), { -> s:Markdown.apply() })
-    endif
+    call s:window.close()
+    call s:padding.close()
+    silent call s:window.open({
```

Comment:
In my environment, neovim will be leaving the rendering garbage so I added `s:window.close` and `s:padding.close` explicitly in here.
Then the documentation window will be display correctly.

However, this approach occurs a flicker, unfortunately.